dev/format_code.sh

```diff
@@ -182,28 +186,35 @@ xmlsort_failed() {
   exit 7
 }
 echo "Formatting resource files with xmlsort..."
-if [ "$ALL" == "--all" ]; then
-  TRACKED=$(git ls-files -- "**.qrc")
-else
-  # Only files which differ from the master branch
-  TRACKED=$(git diff --name-only master -- "**.qrc")
-fi
-UNTRACKED=$(git ls-files --others --exclude-standard -- "**.qrc")
-for file in $TRACKED $UNTRACKED
+for dir in img/ libs/
```

Comment:
Hm, previously the file search was not restricted to these directories, why restricting it now?

Generally I prefer to format all files in the repository, and only blacklisting those coming from third-parties (that's why we limit *.cpp/*.h files to our own source directories).

Reply:
The problem is that in other case it formats the cmake-build-debug directory too and it is really huge as it contains a lot of file copies already changed. Mainly in CMakeFiles scripts but its same for other files. With this limit the speed of stylecheck/formater is comparably faster (0.5sec vs 30sec measured by eye)

Reply:
Restriction removed.